BOUNTY.md

```diff
@@ -1,24 +1,33 @@
+The core development team is tasked with upgrading and improving the blockchain itself, the web app, and a mobile app. This has shifted the focus of the project away from the MDW, and toward other applications more targeted to widening the spread of adoption and community in the ecosystem.
 
-## Guidelines and Instructions
+And here is where you come in! We have created a bounty program for you to help out with your own contributions. If you want to hack on this codebase, read on!! We appreciate your interest in the project, and your effort into the project.
 
-Issues that have a bounty are marked with two different kinds of labels - A Cyan bounty label along with the bounty amount in its respective color. Participants interested in putting in a PR for the same are requested to do so right away, there is no need to claim the issue or request to be assigned towards the same.
+Issues that have a bounty are marked with two labels. Firstly, a cyan bounty label denotes that an issue has a bounty reward associated with it. Secondly, the amount of the reward in $MUSIC. Should you find that one of the bounty issues falls into your wheelhouse; by all means, submit a PR!
 
-Some general guidelines towards the same are:
+## Contributing
 
-1. The best proposal wins, not the first one. So you'll have to make sure that your fix stands out from the rest.
-2. The Musicoin developers would be happy to assist any person with a given issue, at the official Slack/Discord forums.
-3. To claim your bounty, you're required to sign the Musicoin CLA. This will be presented to you after your PR is merged.
-4. New bounty issues can be submitted at any time, the team will review and assign a bounty if needed.
-5. Teamwork is encouraged. If two people work together to fix an issue, both will be rewarded in proportion to their contribution.
-6. General guidelines like DRY and comments are strongly encouraged.
+1. Quality is preferred over turnover time; naturally. So put some effort in, and make your patch awesome.
+2. We would be more than happy to lend assistance if need be. You can find us at the official Slack or Discord.
+3. To claim your bounty, you're required to sign the Musicoin CLA. This will be presented to you if your PR is merged.
+4. New issues can be submitted at any time, the team will review and assign a bounty if needed.
+5. Teamwork is encouraged. If two people work together to fix an issue, both will be rewarded in proportion to their contribution. So pool your skills, and maybe you'll come up with something we didn't.
+6. Comment and fully document your work! This isn't to impose more hoops for you to jump though, but is in the interest of the others who may code on that file next. Make sure to attempt a solid explanation of anything that is not reasonably intuitive.
```

Comment:
`This isn't to impose more hoops for you to jump though, but is in the interest of the others who may code on that file next` is not needed.